exampleCourse/questions/downloadFile/server.py

```diff
@@ -0,0 +1,16 @@
+import matplotlib.pyplot as plt
+import io
+
+def file(data):
+    if data['filename']=='data.txt':
+        return 'This data is generated by code.'
+    elif data['filename']=='figure.png':
+        plt.plot([1,2,3],[3,4,-2])
+        buf = io.BytesIO()
+        plt.savefig(buf,format='png')
+        return buf
+    else:
```

Comment:
Same comment here as above, can we handle `None` to remove the need for this `else:`?